docfx.json

```
@@ -0,0 +1,39 @@
+{
```

You probably didn't mean to include this file as part of the PR ?
```suggestion
{remove this file}
```

---

Yeah, this file was given to me by Bernadette. She said that it should be added to the MRTK root dir.

---

Ok. If we (reasonably) can we should try to separate the commits for the azure pipeline and those for generating documentation.